backend/nginx/conf/nginx-docker.conf

```diff
@@ -20,6 +20,8 @@ events {
 http {
     include       mime.types;
     default_type  application/octet-stream;
+    
+    client_body_temp_path /dcef/cache/nginx/body
```

Comment:
I think that `/tmp/` directory is better for this purpose. It is temporary data, not meant to be persistent. Also missing required `;` at the end.

For example:
```conf
client_body_temp_path /tmp/nginx_client_body 1 2;
```

Reply:
Force pushed with your recommended change, Thanks!